### PR TITLE
Enable Designate (DNS-as-a-Service)

### PR DESCRIPTION
**Pull request by yankcrime**
This commit includes the minimum configuration necessary to enable a deployment of Designate.

### CODE REVIEW

File: etc/kayobe/inventory/group_vars/controllers/network-interfaces [hunk 1]
```diff
@@ -29,6 +29,8 @@ alaska_bdn_interface: brhte
 alaska_bdn_bridge_ports:
   - enp3s0f1
 
+dns_interface: "{{ ilab_interface }}"
```
Comment (markgoddard) [inline etc/kayobe/inventory/group_vars/controllers/network-interfaces]: I don't think this is necessary. Kayobe uses the `public` network for DNS, and passes the interface on that network through to kolla-ansible as `dns_interface`.

Comment (markgoddard) [reply etc/kayobe/inventory/group_vars/controllers/network-interfaces]: The docs probably need a sentence [here](https://kayobe.readthedocs.io/en/latest/configuration/network.html#network-role-configuration) to say that the public network is used for DNS.

Comment (yankcrime) [reply etc/kayobe/inventory/group_vars/controllers/network-interfaces]: Good point, thanks.

File: etc/kayobe/kolla/config/neutron.conf [hunk 2]
```diff
@@ -1,4 +1,4 @@
 [DEFAULT]
-dns_domain = alaskalocal.
+dns_domain = alaska.local.
```
Comment (markgoddard) [inline etc/kayobe/kolla/config/neutron.conf:2]: As discussed at the summit, need to differentiate between prod & alt-1 in the domain name.

**Comment by markgoddard** [comment]
Haven't you finished this yet @yankcrime ?

**Comment by yankcrime** [comment]
@markgoddard No, I'm getting lazy in my old age 😔

**Pull request closed without merging**